RNWCPP/ReactUWP/Executors/WebSocketJSExecutorUwp.cpp

```diff
@@ -213,12 +213,17 @@ winrt::Windows::Foundation::IAsyncAction WebSocketJSExecutor::ConnectAsync(const
 
   SetState(State::Connected);
 
-  PrepareJavaScriptRuntime();
-
-  SetState(State::Running);
+  if (PrepareJavaScriptRuntime())
+  {
+    SetState(State::Running);
+  }
+  else
+  {
+    OnHitError("Create a new JS runtime timeout and Executor instance is not connected to a WebSocket endpoint.");
```

Comment:
Can we make this string a bit more readable for when it shows up in the redbox?  slight adjustment of maybe:
Prepare JS runtime timed out, Executor instance is not connected to a WebSocket endpoint.

Reply:
```suggestion
    OnHitError("Prepare JS runtime timed out, Executor instance is not connected to a WebSocket endpoint.");
```